applications/launchpad/backend/src/docker/workspace.rs

```diff
@@ -466,44 +457,32 @@ impl TariWorkspace {
 
     /// Stop the container with the given `name` and optionally delete it
     pub async fn stop_container(&mut self, name: &str, delete: bool, docker: &Docker) {
-        let container = match self.container_mut(name) {
-            Some(c) => c,
-            None => {
-                info!(
-                    "Cannot stop container {}. It was not found in the current workspace",
-                    name
-                );
-                return;
-            },
-        };
-        let options = StopContainerOptions { t: 0 };
-        let id = container.id().clone();
-        match docker.stop_container(id.as_str(), Some(options)).await {
-            Ok(_res) => {
-                info!("Container {} stopped", id);
-                container.set_stop();
-            },
-            Err(err) => {
-                warn!("Could not stop container {} due to {}", id, err.to_string());
-            },
-        }
-        // Even if stopping failed (maybe it was already stopped), try and delete it
-        if delete {
-            match docker.remove_container(id.as_str(), None).await {
-                Ok(()) => {
-                    info!("Container {} deleted", id);
-                    container.set_deleted();
+        let about_to_delete = CONTAINERS.write().unwrap().remove(name);
+        if about_to_delete.is_some() {
```

Comment:
You don't log the message if we try and stop a non-existent container any more. To keep the same level of functionality you need to handle the `None` branch here and log the message from lines 472-475 on the left

Reply:
Also prefer this form
```
if let Some(container) = about_to_delete {
 // No need for: let container = about_to_delete.unwrap();
 ...
```